3p/frame.max.html

```diff
@@ -2,7 +2,7 @@
 <head>
 <meta charset="utf-8">
 <meta name="robots" content="noindex">
-<script src="./integration.js"></script>
+<script src="./f.js"></script>
```

Comment:
@jrf0110 [line #L115 in 3p-frame.js](https://github.com/ampproject/amphtml/blob/master/src/3p-frame.js) does roughly what your suggesting. The `src` is is built in `getDefaultBootstrapBaseUrl` and passed through `getBootstrapBaseUrl` before appending json. The following could work too.

```
const baseUrl = getBootstrapBaseUrl(parentWindow);
attributes.AMP_CONFIG = urls; 
const src = baseUrl + '#' + JSON.stringify(attributes);
```